app/assets/javascripts/adminComments.js

```diff
@@ -0,0 +1,77 @@
+/* global alert */
+
+var adminComments = (function() {
+  var adminCommentFormSelector = '#new_admin_comment';
+  var adminCommentListSelector = '[data-behavior="admin-comments-list"]';
+  var adminCommentToggleSelector = '[data-behavior="toggle-admin-comment-form"]';
+  var adminCommentFormCancelButton = '[data-behavior="admin-comment-cancel"]';
+
+  function commentTemplate(comment) {
+    return '<li>' +
+             comment.comment +
+             '<span class="text-muted">' +
+               ' - ' +
+               comment.commenter + ' - ' +
+               comment.created_at +
+              '</span>' +
+           '</li>';
+  }
+
+  return {
+    init: function(holdingsTable) {
+      this.holdingsTable = $(holdingsTable);
+      this.form = this.holdingsTable.find(adminCommentFormSelector);
+
+      this.addCommentToggleBehavior();
+
+      this.onAjaxSuccess();
+
+      this.onAjaxError();
+
+      this.addCancelButtonBehavior();
+    },
+
+    addCommentToggleBehavior: function() {
+      var _this = this;
+      var commentToggle = _this.holdingsTable.find(adminCommentToggleSelector);
+      commentToggle.on('click', function() {
+        _this.form.slideToggle();
```

Comment:
D'oh! Totally what I was intending by moving that into its own function.  Good catch!